udata/core/dataset/api.py

```diff
@@ -288,12 +288,13 @@ def post(self, dataset):
 @api.doc(parser=upload_parser, **common_doc)
 class UploadCommunityResources(UploadMixin, API):
```

Comment:
Alright, can we at least rename the classes to be more explicit or at least add a comment related to that?